test/isochrone.cc

```diff
@@ -355,7 +355,6 @@ void check_raster_edges(size_t x, size_t y, uint16_t* data) {
 }
 
 TEST(Isochrones, test_geotiff_output_distance) {
-  GDALRegister_GTiff();
```

Comment:
i dont really like removing these because the tests themselves use gdal directly to load the response from the library. now in this case technically we can remove them because hidden in the source of the serializer we are doing this initialization but the tests (and just in general any calling code) shouldnt know/assume that its preconditions are met by something it calls into. the only way to get around this would be to expose gdal initialization as a public api in the library and have the tests call that but i hate that idea even worse.

Reply:
Ah I didn’t check that.. I kinda assumed those driver inits were there to make the test work (it wouldn’t have without). That’s true.. anyways, the alternative to test this bit is indeed worse.

Reply:
the other idea is to leave them there though what i would do is in the suite initialization call it once

Reply:
Both is fine with me tbh, I get your point, but also think it’ll be fine probably forever ®